### DISCUSSION TITLE
🚀 New Theme Announcement: ZedSpace

### DISCUSSION THREAD

**Discussion by brunowilliang** [zed-industries/zed#7773] [Show and tell]
I'm thrilled to share with you a new theme I've created.
Designed with a focus on readability, aesthetics, and productivity.

https://github.com/Brunowilliang/zedspace

<h3 align="center">
	<a href="https://github.com/Brunowilliang/zedspace">ZedSpace</a> for <a href="https://zed.dev/">Zed</a>
</h3>

![preview](https://github.com/zed-industries/zed/assets/41432581/1818f1cb-0f78-48fc-8b9b-d39a6b452466)

**Comment by JosephTLyons** [comment]
Hey @Brunowilliang, this looks fantastic! Would love for you to add it to our extension store:

- https://github.com/zed-industries/extensions

**Reply by brunowilliang** [reply]
Hey @JosephTLyons I created a pull request to add.
